src/App.vue

```diff
@@ -414,7 +414,23 @@ export default class App extends Mixins(CommonMixin, FilingTemplateMixin) {
 
     // ensure user is authorized to access this business
     try {
-      await this.checkAuth()
+      // get roles from KC token
+      const authRoles = GetKeycloakRoles()
+
+      // safety check
+      if (!Array.isArray(authRoles)) {
+        throw new Error('Invalid roles')
+      }
+
+      // verify that response has one of the supported roles
+      // FUTURE: when we fetch authorized actions from Legal API, we'll instead check
+      //         that the list of actions isn't empty
+      const allRoles = Object.values(AuthorizationRoles)
+      if (!allRoles.some(role => authRoles.includes(role))) {
+        throw new Error('Missing valid role')
+      }
+
+      this.setAuthRoles(authRoles)
```

Comment:
I didn't bother renaming the store property/setter/getter to "Keycloak Roles", but this will be refactored in a future ticket anyway (where we'll fetch authorized actions directly).